modules/tensor_mechanics/tests/tensile/tests

```diff
@@ -81,6 +81,46 @@
     abs_zero = 1.0E-5
     cli_args = '--no-trap-fpe'
   [../]
+  [./planar4]
+    type = 'CSVDiff'
+    input = 'planar4.i'
+    csvdiff = 'planar4.csv'
+    abs_zero = 1.0E-8
+    rel_err = 1.0E-5
+    cli_args = '--no-trap-fpe'
```

Comment:
Are you actually getting underruns or overruns in these tests so that you need this flag? Just curious